Lib/javascript/v8/javascriptrun.swg

```diff
@@ -466,7 +477,7 @@ SWIGRUNTIME SWIGV8_VALUE SWIG_V8_NewPointerObj(void *ptr, swig_type_info *info,
 
 #define SWIG_GetInstancePtr(obj, ptr)    SWIG_V8_GetInstancePtr(obj, ptr)
 
-SWIGRUNTIME SwigV8ReturnValue _SWIGV8_wrap_equals(const SwigV8Arguments &args) {
+SWIGRUNTIME SwigV8ReturnValue _wrap_SWIGV8_Proxy_equals(const SwigV8Arguments &args) {
```

Comment:
_wrap prefix is reserved for wrapper code, so best left as it was.

Reply:
I've spotted inconsistency in `_SWIGV8_wrap_equals` vs. `_wrap_getCPtr` naming and tried to harmonize the names. Rationale behind switching is that these are methods just like any other. They are kind of hardwired. But sure, let's stick with `_SWIGV8_wrap_equals`. Would it be appropriate to rename `_wrap_getCPtr` as `SWIGV8_wrap_getCPtr`?

Reply:
This looks bit of a mess in the Javascript module, both v8 and jsc. There are a number of runtime functions (the ones using the `SWIGRUNTIME` macro) which should begin with `Swig` or `SWIG`, not `_wrap`. In fact, all of the functions in the Lib/javascript directory should begin not begin with `_wrap`. I would be more than happy to consider a separate pull request to fix all of these.

Reply:
I've removed the method renaming part.